pages/_document.tsx

```diff
@@ -1,4 +1,4 @@
-import { ServerStyleSheets as ServerStyleSheetsMUI } from "@material-ui/core";
+import { ServerStyleSheets as ServerStyleSheetsMUI } from "@mui/styles";
```

Comment:
Site fails to compile. 
Module not found: Can't resolve '@mui/styles'

Reply:
try yarn install. if that doesn't work, delete node module and package.lock and then try yarn install again

Reply:
Never mind this, it was fixed with a "yarn install".
The rest of the PR looks good, however this request for change will still remain as there's still 1 code smell that needs to be fixed.